### PR TITLE
#863 history button location

### LINKED ISSUE #863: History button location
Find consistent location in the UI for the history buttons

### PR DESCRIPTION
**Pull request by roxy-dao**
Closes #863 

- Unified the `History` button by adding it to the Footer on the Patient view. 
- Added `History` to Program Enrolments in tab view

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -71,6 +71,8 @@
   "heading.settings-display": "Display settings",
   "heading.settings-sync": "Synchronisation",
   "heading.shipments-to-be-picked": "Shipments to be picked",
+  "heading.details": "Details",
```
Comment (roxy-dao) [inline client/packages/common/src/intl/locales/en/common.json:74]: Should this be in patient.json instead...? 🤔

Comment (clemens-msupply) [reply client/packages/common/src/intl/locales/en/common.json:74]: Seems generic enough to have it here. Or reuse the "label.details": "Details" from patient.json ?

**Review by clemens-msupply — APPROVED** [review]
Looks good, thanks! could you force the program modal to have the full height so that it doesn't jump when switching tabs?

File: client/packages/common/src/intl/locales/en/common.json (hunk 1 above)
Comment (roxy-dao) [reply client/packages/common/src/intl/locales/en/common.json:74]: Maybe will leave it here if it's generic enough 🤔

**Pull request merged**